Include/pyexpat.h

```diff
@@ -48,6 +52,9 @@ struct PyExpat_CAPI
     enum XML_Status (*SetEncoding)(XML_Parser parser, const XML_Char *encoding);
     int (*DefaultUnknownEncodingHandler)(
         void *encodingHandlerData, const XML_Char *name, XML_Encoding *info);
+#if PYEXPAT_COMBINED_VERSION >= 20100
```

Comment:
Couldn't adding the slot conditionally break binary compatibility?

Reply:
The new PyExpat_CAPI_MAGIC version ensures that only compatible modules are loaded.